app.py

```diff
@@ -31,23 +36,72 @@ def healthcheck():
 def login():
     '''
     Login endpoint to authenticate the user
+
+    Expects:
+        JSON body with 'username' and 'password'
     
     Returns:
         JSON response indicating the status of the login
     '''
-    pass
+    data = request.get_json()
+    username = data.get('username')
+    password = data.get('password')
+
+    user = user_model.find_user_by_user(username)
+    if user and user.password == password:
```

Comment:
I think we need to hash our passwords on the db side, we can use the [bcrypt](https://pypi.org/project/bcrypt/) library for this. Make sure we add it to requirements.txt and requirements.lock as well